Sprint-2/interpret/time-format.js

```diff
@@ -29,3 +29,6 @@ function formatTimeDisplay(seconds) {
 // d) What is the value assigned to num when pad is called for the last time in this program?  Explain your answer
 
 // e) What is the return value assigned to num when pad is called for the last time in this program?  Explain your answer
+
```

Comment:
#### Todo
In this exercise, you need to answer questions (a)-(e) by tracing the execution of the code (that is, figure out in what order every statement and every function call is executed) .